src/jit/compiler.h

```diff
@@ -517,7 +517,7 @@ class LclVarDsc
     {
         if (slotNum == 0)
         {
-            return lvArgReg;
+            return GetArgReg();
```

Comment:
Nit: it could be `_lvArgReg` because it is inside `LclVarDsc` and we don't usually use getters/setters in member functions, but I think it does not matter much.